src/web/views/Content/index.jsx

```diff
@@ -265,3 +185,15 @@ export default class ContentView extends Component {
     )
   }
 }
+
+const mapStateToProps = state => ({
+  categories: state.content.categories,
+  messages: state.content.currentMessages
+})
+const mapDispatchToProps = dispatch =>
+  bindActionCreators(
+    { fetchContentCategories, fetchContentMessages, upsertContentMessages, deleteContentMessages },
```

Comment:
I think here and everywhere you can just use the plain object, Redux will bind actions for you:

```
const mapDispatchToProps = { fetchContentCategories, fetchContentMessages, upsertContentMessages, deleteContentMessages }
```

Reply:
Good point! Done!